app/utils/feature-flag.js

```diff
@@ -7,4 +7,11 @@ function enableVlaamsParlement() {
   );
 }
 
-export { enableVlaamsParlement };
\ No newline at end of file
+function enableNewCaseCreation() {
```

Comment:
Perhaps rename to `isEnabledNewCaseCreation`?

`enableXXX` makes it sound like the method will turn on the feature/flag.